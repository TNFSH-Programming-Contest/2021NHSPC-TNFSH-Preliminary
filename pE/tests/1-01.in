1001 1000
3 4 13
4 5 23
6 3 25
7 6 23
5 8 21
8 9 22
10 7 7
9 11 14
12 13 25
14 12 19
15 10 15
13 15 23
16 14 9
17 16 12
18 17 10
11 19 12
20 21 21
22 23 29
21 24 6
24 22 11
25 26 18
23 27 21
28 29 7
30 25 18
31 32 30
27 33 29
33 34 28
35 30 18
36 18 13
37 20 17
32 36 25
38 39 8
39 40 19
34 41 6
42 35 24
43 42 28
44 45 6
46 37 7
19 44 24
47 48 16
29 49 18
50 31 13
49 50 10
51 52 27
45 53 27
53 54 6
41 55 12
55 43 28
56 47 29
57 58 15
54 59 12
59 60 19
26 51 26
61 28 9
60 62 17
63 56 27
40 64 15
65 63 13
64 66 18
67 68 23
69 70 22
71 72 30
66 73 26
48 57 23
62 74 16
73 67 15
75 76 30
77 69 5
78 79 9
80 65 15
79 81 30
82 83 18
52 38 29
84 85 14
72 86 26
87 71 12
88 89 10
70 90 10
91 92 9
93 94 9
94 95 26
68 96 18
86 93 20
95 97 8
92 87 21
85 77 18
89 80 17
98 99 15
81 84 27
97 100 14
96 101 12
76 102 10
103 46 25
58 103 27
104 105 10
106 107 7
101 104 10
105 61 27
108 109 12
102 110 14
83 75 19
111 108 20
112 106 18
113 82 24
114 115 22
116 111 7
109 117 30
90 118 26
107 113 30
99 112 17
119 78 15
118 98 18
120 116 19
110 120 20
121 119 8
122 88 7
123 124 12
125 126 27
117 127 6
126 128 11
128 121 12
74 91 21
129 114 20
130 131 22
132 133 10
124 129 22
100 134 5
127 135 22
115 136 18
131 137 19
138 125 16
139 122 22
137 140 16
141 142 30
143 123 9
140 144 16
145 141 16
146 143 23
142 138 8
147 148 6
148 149 7
135 139 24
150 147 8
134 151 24
136 152 22
152 153 6
154 145 27
149 132 28
133 155 14
155 156 28
157 154 12
144 158 27
159 150 19
160 161 12
162 163 8
153 164 7
163 157 24
165 166 25
167 168 5
158 169 17
168 170 18
164 171 25
172 173 15
174 160 8
175 176 26
177 178 20
179 180 27
171 174 17
181 177 11
161 130 24
180 182 5
183 175 11
184 185 24
186 165 23
182 146 22
187 188 18
178 189 11
190 179 19
169 172 24
191 183 17
156 181 24
173 192 21
185 187 18
188 191 16
193 190 23
194 193 25
195 167 12
196 186 25
189 195 14
151 197 29
198 199 13
200 159 29
201 162 30
202 184 14
203 202 23
170 201 12
204 205 21
206 207 9
192 208 14
166 200 20
197 209 11
210 211 27
212 213 29
209 214 7
199 215 7
176 216 26
217 218 23
216 219 29
215 196 5
220 221 22
222 223 17
224 225 20
205 226 11
218 227 5
228 220 19
226 229 23
211 203 6
230 231 6
232 194 25
219 233 20
234 235 25
208 236 8
237 212 8
221 234 16
238 222 30
235 239 18
240 241 6
241 237 18
242 198 23
243 238 30
227 228 18
244 245 28
223 246 28
239 204 26
207 247 26
236 248 14
249 206 22
250 232 20
251 252 23
253 224 30
254 255 23
256 257 19
247 244 27
258 251 17
259 260 12
231 261 18
262 263 29
260 264 30
246 217 25
264 265 22
266 242 5
267 268 10
269 258 8
252 256 27
225 270 12
271 254 22
272 249 20
273 274 29
265 275 21
213 266 17
276 210 10
233 262 28
277 278 18
270 273 24
279 272 5
280 281 23
278 269 24
245 250 29
282 283 23
284 230 26
285 286 29
287 253 28
275 243 21
288 282 5
281 289 22
290 287 25
229 291 11
261 292 7
274 293 16
293 294 12
295 288 26
291 284 30
296 297 30
263 298 19
299 300 14
301 285 11
302 295 6
298 267 18
303 299 6
214 304 20
305 306 19
292 307 9
255 302 6
308 309 22
248 305 25
310 311 5
311 240 13
312 313 18
306 280 30
314 312 27
304 296 22
268 259 9
315 316 5
297 317 27
318 319 13
289 320 25
319 321 17
322 301 20
323 324 22
307 303 21
313 290 7
317 325 24
326 327 8
300 318 12
286 310 8
320 328 5
283 323 10
329 330 21
321 277 5
257 326 19
316 329 6
309 331 8
332 314 18
333 315 8
334 333 7
335 271 30
336 337 6
338 339 17
330 338 8
340 341 19
337 332 26
339 342 21
294 335 20
343 340 25
342 344 15
325 345 5
346 279 8
341 347 17
347 308 29
331 334 11
344 348 14
348 276 17
327 346 17
345 349 18
349 350 12
324 343 10
350 351 12
328 322 26
351 352 6
353 354 6
355 353 23
356 355 16
354 357 19
357 358 10
359 336 9
360 356 30
358 359 12
361 360 7
362 361 26
363 364 13
364 362 10
352 365 8
365 366 28
367 368 16
369 367 27
370 371 10
366 372 14
373 374 13
375 376 16
376 370 9
368 377 17
378 363 17
379 380 5
381 379 30
382 381 13
383 382 5
374 375 7
384 385 8
377 373 12
372 383 19
371 386 13
387 384 28
388 389 30
386 390 11
391 392 26
393 391 14
394 388 21
392 387 5
395 378 20
396 393 10
397 398 30
399 400 12
401 394 13
402 369 5
389 399 12
403 404 18
405 406 5
406 407 24
385 408 28
380 401 27
409 410 23
411 412 22
413 409 19
398 414 20
414 415 16
416 402 7
404 417 26
418 419 9
408 416 11
415 420 22
421 422 7
423 424 22
400 413 12
425 426 10
417 418 6
427 428 9
407 425 16
429 405 6
430 431 19
431 432 28
390 427 10
410 430 19
419 433 13
422 395 11
434 435 5
436 437 21
437 438 15
432 439 5
428 440 30
420 436 29
426 403 10
424 434 12
441 397 5
442 421 26
443 444 16
445 442 13
446 447 27
440 446 28
448 443 28
435 429 16
439 396 13
433 449 29
444 450 6
449 411 17
450 423 29
451 452 19
412 441 22
453 445 17
452 454 10
438 451 17
455 456 29
457 458 8
459 460 25
460 461 8
462 459 23
463 462 30
464 448 20
465 466 7
454 467 21
468 469 17
470 471 20
472 465 20
473 474 5
475 463 28
471 476 14
474 477 27
447 468 20
478 472 19
479 480 29
481 482 12
483 484 7
485 478 23
486 453 22
466 470 15
476 487 10
488 489 23
490 455 10
480 491 8
492 473 15
484 490 28
469 493 30
487 494 27
491 475 29
477 495 21
496 488 7
497 486 20
494 481 27
498 499 12
500 501 9
502 503 9
499 485 15
458 498 11
504 505 13
461 457 23
495 506 14
501 504 26
467 500 28
507 479 23
508 502 19
456 492 21
509 510 25
511 512 11
513 508 18
493 514 20
510 483 13
482 511 21
503 515 19
516 497 16
517 513 18
512 496 5
518 464 26
506 519 7
520 521 16
521 518 18
522 523 6
524 525 11
526 524 8
527 517 8
528 529 24
489 530 10
531 527 28
532 533 17
534 535 12
536 537 18
538 539 26
539 540 8
541 538 25
535 542 27
515 543 15
514 528 20
544 545 21
546 547 14
519 544 19
548 509 20
523 507 17
549 550 19
551 552 9
533 526 28
553 548 5
554 555 13
530 532 17
556 551 20
543 553 26
537 554 5
557 558 11
540 546 25
559 531 23
560 561 14
547 557 10
562 563 23
564 565 15
566 541 9
545 567 26
558 522 19
529 568 9
568 556 29
542 569 14
570 564 26
571 566 22
572 573 20
552 574 5
575 576 11
561 577 26
578 579 25
525 578 22
550 572 20
574 520 19
580 562 18
581 571 23
579 582 28
583 584 17
585 586 29
587 588 13
589 590 20
577 591 25
592 593 30
594 534 28
595 570 15
591 583 21
596 560 24
567 581 17
597 585 30
582 592 12
598 580 7
599 559 7
600 601 26
555 602 24
590 603 23
601 598 28
604 516 10
569 605 21
563 604 7
606 536 9
505 606 22
607 599 9
586 608 22
588 609 24
610 611 29
612 589 5
573 613 22
614 612 23
615 607 25
611 549 25
613 616 28
565 617 18
618 595 21
619 620 30
609 597 19
602 575 5
605 621 22
622 619 6
623 624 28
625 614 21
626 627 19
628 629 13
584 630 9
621 625 30
631 632 5
633 618 24
634 622 10
629 594 11
624 635 26
636 637 10
632 638 15
620 639 15
640 641 11
608 596 24
642 643 13
644 645 5
646 644 11
616 615 15
638 640 21
641 628 24
647 648 22
649 650 29
651 623 27
645 600 26
637 652 23
653 654 6
627 655 25
630 656 29
654 657 6
639 653 5
650 631 6
617 658 13
659 660 7
655 647 9
576 661 8
662 663 19
664 665 24
660 666 5
663 667 6
668 669 16
669 670 24
671 587 8
672 659 13
652 610 27
670 673 28
665 642 27
635 672 19
674 636 18
656 675 6
675 676 21
657 646 30
677 678 20
679 634 29
603 680 27
666 649 18
681 682 18
683 684 25
685 679 17
686 633 6
687 651 8
667 688 27
689 681 13
680 662 23
678 690 12
691 689 5
692 693 23
694 626 26
661 695 5
658 696 24
697 698 8
688 699 7
593 671 21
700 674 7
701 702 22
703 686 23
704 705 14
706 677 9
707 708 8
696 709 30
698 701 23
710 711 17
705 712 19
690 713 19
714 706 19
715 707 6
673 697 6
716 717 30
718 714 16
719 715 12
720 719 29
682 704 13
721 722 10
648 723 21
643 724 30
725 726 20
727 664 14
728 668 23
729 691 27
730 731 29
732 733 10
733 730 8
734 735 26
736 700 20
735 727 16
695 728 24
737 738 23
717 739 7
708 716 7
740 741 15
712 742 7
743 744 11
723 718 15
693 745 16
744 720 11
746 725 5
747 746 11
684 732 12
742 748 28
749 737 6
745 683 23
750 751 11
752 694 15
702 687 5
739 753 8
754 755 8
756 757 21
758 729 14
759 747 9
760 761 11
762 763 22
731 685 19
764 765 25
766 762 7
753 767 5
722 734 18
763 692 6
709 760 15
768 736 22
769 740 8
757 721 8
751 770 16
771 772 10
741 768 7
773 703 8
774 775 22
748 764 12
724 776 6
777 778 19
778 779 22
780 777 18
781 782 13
783 784 11
713 785 16
755 786 11
787 788 13
789 710 7
765 769 14
726 787 8
772 766 21
788 773 23
790 791 24
761 756 24
792 749 25
784 752 5
793 789 22
779 794 23
738 795 23
796 771 26
797 798 29
799 758 16
676 800 13
801 774 13
785 743 26
794 802 20
795 803 26
802 801 26
804 805 10
776 806 22
775 807 13
808 809 15
767 810 9
803 780 7
770 793 23
811 812 22
807 783 29
711 813 24
814 815 5
805 816 17
817 818 5
782 819 17
820 821 8
822 823 12
816 824 18
815 825 19
826 827 29
827 804 11
828 829 20
810 781 18
830 831 7
823 808 26
798 750 25
831 826 29
806 832 10
833 834 10
813 817 22
818 835 17
699 836 12
819 837 29
838 792 17
821 839 9
786 828 10
840 841 9
842 833 15
837 843 6
844 814 22
845 846 12
829 759 16
846 847 21
847 790 7
848 820 23
849 850 12
835 851 17
852 842 11
853 799 21
854 852 15
855 856 27
857 858 26
859 848 18
800 849 15
809 845 28
860 822 11
861 855 15
850 862 12
863 844 7
812 864 23
865 830 5
858 860 26
843 866 7
867 868 13
869 870 22
825 796 28
864 853 30
866 754 15
871 869 7
872 857 26
839 873 15
824 874 18
875 876 25
876 877 14
851 878 19
879 880 13
832 865 11
873 838 23
881 882 30
874 797 28
883 854 26
877 861 11
836 872 12
868 884 7
841 885 8
886 840 19
887 888 6
889 890 18
834 867 28
891 811 23
892 871 5
890 881 10
882 893 19
894 859 14
884 895 22
896 897 23
898 899 10
862 900 28
901 902 30
870 863 22
878 903 28
903 883 15
880 889 26
902 904 12
791 887 7
905 906 13
907 908 6
888 909 20
910 911 18
912 879 14
913 875 20
914 901 9
899 912 10
915 916 5
917 918 5
919 907 12
920 921 30
885 910 27
856 922 7
906 896 21
908 923 23
904 891 30
924 917 22
925 919 24
926 927 17
897 915 14
928 929 25
893 930 8
918 931 12
932 886 14
933 892 25
934 935 5
921 905 7
900 898 14
909 936 28
916 937 9
922 933 19
938 939 15
940 920 20
929 941 30
930 942 26
943 944 5
945 926 15
931 925 26
946 947 18
942 894 7
948 932 20
949 940 19
937 950 5
911 951 20
936 952 13
953 913 15
923 954 30
955 945 26
950 924 20
951 956 18
957 958 28
958 959 26
960 961 8
935 948 17
939 962 15
952 963 27
964 965 25
966 938 7
967 968 11
947 934 30
895 969 18
970 971 20
927 966 14
965 970 6
972 964 30
973 974 28
963 928 28
962 967 9
971 914 17
975 976 23
954 977 29
944 978 26
979 949 5
980 960 13
981 982 25
983 955 22
969 975 30
984 972 14
976 979 24
982 985 10
986 946 13
968 987 9
988 984 19
977 973 7
989 990 5
991 980 26
941 986 8
974 991 21
1 992 13
956 993 23
994 995 13
961 996 29
997 983 7
959 994 30
998 999 5
990 981 12
996 943 20
985 1000 28
992 988 17
999 997 12
978 953 22
993 989 23
987 2 15
1001 957 15
995 998 10
1000 1001 26
