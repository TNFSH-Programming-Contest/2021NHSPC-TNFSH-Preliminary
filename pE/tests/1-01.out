17326.0000000000000000000
